### ISSUE TITLE
Make LUA value keeps losing the type

### ISSUE THREAD

**Issue by DoubleP90** [rdeioris/LuaMachine#32]
Make Lua value keeps losing the type, requiring a refresh of the node
![Type recompile](https://user-images.githubusercontent.com/7040909/130336400-5fb1757f-f490-4af0-8741-27ff8a53040c.jpg)

After refreshing the node I lose the type enum 
![Refreshed](https://user-images.githubusercontent.com/7040909/130336405-458a6b99-5ae5-47a5-8c4c-1bf5facd6f75.jpg)

After a while I am able to refresh the node again and the enum type returns
![Returned](https://user-images.githubusercontent.com/7040909/130336428-7133253d-db2a-4d6e-977f-5a87de3dd012.jpg)

Sometimes the warning returns and I lose the enum again

This happens every time I start the project, the issue is present in 4.26 (where the screenshots are from, no source modification) and 5.0

**Comment by Olathuss** [comment]
I don't think this is a problem with LuaMachine, most likely it's Unreal. Are you compiling the plugin from source? That's the only issue I can think which would cause this, because Unreal thinks the struct has changed for some reason. Recompiling the source typically causes this, although it should only happen if you changed the Plugin source.

There are two potential workarounds:
Use the Struct's Constructor method instead of Unreal's "Make (Struct)" operator:
![image](https://user-images.githubusercontent.com/1407097/234147660-a771bfcb-cf00-4e47-b2a7-80723fc9baae.png)

It will automatically set the type for you.

Or check which pins you have visible on the details panel:
![image](https://user-images.githubusercontent.com/1407097/234148675-13c71e09-eefd-4717-b6d5-135d629ceadc.png)

**Comment by Olathuss** [comment]
I just tried this on a 5.1 project and I'm not able to replicate it.
What type of Blueprint are you using? Actor, LuaState, Component?
